packages/rx_bloc/lib/src/result_extensions.dart

```diff
@@ -44,41 +46,41 @@ extension ResultMapStreamX<E> on Stream<E> {
 
 extension ResultMapStream<E> on Stream<Result<E>> {
   /// Map the current [Result] stream to a new [Result] one,
-  /// which could by from different type.
+  /// which could be from a different type.
   ///
   /// Use the [mapSuccess] to map the success from the current stream to the
   /// result stream.
   /// Example:
   /// ```
   /// Result<String> result = Stream.value(Result.success(10))
-  ///   .mapResult((value) => value * 10)
-  ///   .mapResult((value) => value.toString()); // Result.success('100')
+  ///   .mapResultStream((value) => value * 10)
```

Comment:
There is no need of the extra suffix `Stream`, so please keep the shorter version `mapResult`